client/src/test/scala/org/http4s/client/ClientRouteTestBattery.scala

```diff
@@ -123,14 +124,12 @@ abstract class ClientRouteTestBattery(name: String, client: Client[IO])
 
     val os: ServletOutputStream = srv.getOutputStream
 
-    val writeBody: IO[Unit] = resp.body
-      .evalMap { byte =>
-        IO(os.write(Array(byte)))
-      }
+    resp.body.through(writeOutputStream[IO](IO.pure(os)))
```

Comment:
os.flush() will throw an exception I believe since `writeOutputStream` closes the output stream after use.

Reply:
Nope.  But we shouldn't close what we don't open, and a servlet should flush when _it_ closes.  So I simplified it some more.